provider/tencentcloud/cloudapi/tencentapi.go

```diff
@@ -258,16 +258,16 @@ func dealWithError(action Action, request string, err error) bool {
 }
 
 func APIErrorRecord(apiAction Action, request string, response string, err error) {
-	log.Infof(fmt.Sprintf("APIError API: %s/%s Request: %s, Response: %s, Error: %s", apiAction.Service, apiAction.Name, request, response, err.Error()))
+	log.Infof("APIError API: %s/%s Request: %s, Response: %s, Error: %s", apiAction.Service, apiAction.Name, request, response, err.Error())
```

Comment:
Same as the previous comment. I will skip other comments for now.

Reply:
In this case, linter says that we should avoid call _Infof_ without args. same here: the resulting code is the same.